tests/conftest.py

```diff
@@ -1906,6 +1909,25 @@ def context_config(
             },
             "tower_workflow": "raredisease",
         },
+        "tomte": {
+            "binary_path": Path("path", "to", "bin", "nextflow").as_posix(),
```

Comment:
```suggestion
            "binary_path": Path("path", "to", "bin", "nextflow").as_posix(),
```
this is repeating under `rnafusion`. Maybe there are others and could warrant fixtures to keep it DRY

Reply:
added. I really want to tackle the mess with the fixtures in another PR. But changed these ones for now